app/models/ApprovalProcessReview.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import java.time.LocalDate
+
+import play.api.libs.json.{Json, OFormat}
+
+case class ApprovalProcessReview(id: String, title: String, lastUpdated: LocalDate, pages: List[PageReview])
+
+object ApprovalProcessReview {
+  implicit val formats: OFormat[ApprovalProcessReview] = Json.format[ApprovalProcessReview]
+}
+
+case class PageReview(id: String, title: String, status: String)
+
+object PageReview {
+  implicit val formats: OFormat[PageReview] = Json.format[PageReview]
+}
```

Comment:
Please move into a `PageReview.scala` file as it makes finding classes easier when not using an IDE e.g. terminal, GitHub UI or file browser.

Also, this is recommended in the [Scala Documentation](https://docs.scala-lang.org/style/files.html) site

Reply:
Done - will be in next push